config/experiment/simple_a2c/cartpole.yaml

```diff
@@ -2,15 +2,15 @@
 defaults:
   - override /services/actor:
       - simple_a2c
-  - override /run: simple_a2c
+  - override /run: headless_play
   - override /services/environment: cartpole
 
 run:
   class_name: actors.simple_a2c.SimpleA2CTraining
   seed: 618
   num_epochs: 500
-  epoch_num_trials: 10
-  num_parallel_trials: 8
+  epoch_num_trials: 1
+  num_parallel_trials: 1
```

Comment:
If it does run in parallel, please use a larger value, if not, please remove it.

Reply:
I've changed to larger values but this choice should be selected by the users on whether if they want to run 
- one trial -> one agent
- multiple trial -> one agent 

The reason is because PPO is on-policy RL algorithm. It is a bit different from DQN i.e., off-policy RL algorithm